app/templates/components/layer-palette.hbs

```diff
@@ -266,7 +266,40 @@
       </ul>
       {{/if}}
     {{/layer-menu-item}}
-    {{layer-menu-item for='aerials-16'}}
+
+    <div 
+      class="layer-menu-item" 
+      {{action 'switchAerial' 16 true}}
+    >
+      {{switch-toggle 
+        checked=(or qps.aerials-16 qps.aerials-1924 qps.aerials-1951 qps.aerials-1996)}}
+      Aerials
+    </div>
+    {{#if (or qps.aerials-16 qps.aerials-1924 qps.aerials-1951 qps.aerials-1996)}}
+      <ul class="layer-menu-item--group-checkboxes">
+        <li {{action 'switchAerial' 16}}>
+          <label>
+            {{fa-icon (if qps.aerials-16 "circle" "circle-o")}} 2016
```

Comment:
Use `circle-thin` and `dot-circle` (as on [PFF](https://github.com/NYCPlanning/labs-nyc-factfinder/blob/68d5ce3736964b6fe53c992637efeb4680d9e736/app/templates/components/profile-header.hbs#L13))

Reply:
I think those are supported in the new version of font awesome and this is still on the old version... will look into upgrading